### PR TITLE
Pan3256

### PR DESCRIPTION
**Pull request by andyb3**
This change is [Reviewable](https://reviewable.io/reviews/moka-guys/mokabed/118)

### CODE REVIEW

File: LiveBedfiles/Pan3256_no_extra_transcripts_LogFile.txt [hunk 1]
```diff
@@ -15,3 +15,15 @@ RefSeq table format version generated as /home/dnanexus/out/Output_files/Pan3256
  cruzdb module file path: /home/dnanexus/anaconda2/lib/python2.7/site-packages/cruzdb/__init__.py
 
 version as defined by git tag = v1.2
+
+
+Use bedtools subtract to add regions from extra transcripts that aren't in main file. Then rename file.
+bedtools subtract -a Pan3256_extra_transcriptsdata.bed  -b Pan3256_no_extra_transcriptsdata.bed >> Pan3256_no_extra_transcriptsdata.bed
+sort Pan3256_no_extra_transcriptsdata.bed -k1,1V -k2,2n -k3,3n > Pan3256_no_extra_transcriptsdata.sorted.bed && mv Pan3256_no_extra_transcriptsdata.sorted.bed Pan3256data.bed
```
Comment (aledj2) [inline LiveBedfiles/Pan3256_no_extra_transcripts_LogFile.txt]: assume sorting hasn't been done yet...

File: LiveBedfiles/Pan3252_LogFile.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
 Rename SNP_ID v2 Pan3252.txt to Pan3252.bed
-git mv 'SNP_ID v2 Pan3252.txt' Pan3252.bed
\ No newline at end of file
+git mv 'SNP_ID v2 Pan3252.txt' Pan3252.bed
+
+Coordinates are in b38, but we need b37, so liftover using UCSC (uploaded bed through web interface, used default settings)
```
Comment (aledj2) [inline LiveBedfiles/Pan3252_LogFile.txt]: assume this was checked?

**Comment by aledj2** [comment]
what a rollercoaster!

File: LiveBedfiles/Pan3256_no_extra_transcripts_LogFile.txt (hunk 1 above)
Comment (aledj2) [reply LiveBedfiles/Pan3256_no_extra_transcripts_LogFile.txt]: can see it was done in a later commit - log file out of sync

**Comment by aledj2** [comment]
files look good but can you also commit the pantranscript files please

**Comment by aledj2** [comment]
also, if refseq file is inaccurate can we delete it?

**Comment by aledj2** [comment]
This review was done via githib, not reviewable. I went through each commit one at a time and (possibly  due to the number of commits) reviewable couldn't support this

**Comment by andyb3** [comment]
> also, if refseq file is inaccurate can we delete it?

Done

**Comment by andyb3** [comment]
> files look good but can you also commit the pantranscript files please

Done

**Pull request merged**